### PR TITLE
fix: Dockerfile

### PR DESCRIPTION
**Pull request by GoliathLabs**
## Checklist:

- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 RUN npm run build
 
-FROM nginx:1.25.3-alpine
+FROM nginx:1.25.4-alpine
 COPY --from=build /code/build/ /usr/share/nginx/html
-COPY --from=build /code/config.json /usr/share/nginx/html/config.json
-
-EXPOSE 80
\ No newline at end of file
+COPY --from=build /code/config*.json /usr/share/nginx/html/
```
Comment (T0biii) [inline Dockerfile]: i think we should copy the [config.example.json](https://github.com/freifunkMUC/meshviewer/blob/main/config.example.json)?

**Comment by T0biii** [comment]
I would now create a new branch from freifunk/main and make the important changes there like
Add Config and Github Action Deploy
I would rather see something like this DockerFile being opened upstream as a PR so that we don't just have the changes with us

following files should be added/adjusted to/for upstream

- https://github.com/freifunkMUC/meshviewer/blob/bbb03481dc2cc2df5201a8322102e3da36a18969/Dockerfile
- https://github.com/freifunkMUC/meshviewer/blob/73eb86bc7e5ec6ca1917ea7cc2efda446d8c123a/.github/workflows/docker.yml
- https://github.com/freifunkMUC/meshviewer/blob/73eb86bc7e5ec6ca1917ea7cc2efda446d8c123a/.github/workflows/build.yml
- https://github.com/freifunkMUC/meshviewer/blob/7f168658a5eef54a2659406744a73fc688243837/docker-compose.yml

**Comment by GoliathLabs** [comment]
> I would now create a new branch from freifunk/main and make the important changes there like Add Config and Github Action Deploy I would rather see something like this DockerFile being opened upstream as a PR so that we don't just have the changes with us
> 
> following files should be added/adjusted to/for upstream
> 
>     * https://github.com/freifunkMUC/meshviewer/blob/bbb03481dc2cc2df5201a8322102e3da36a18969/Dockerfile
> 
>     * https://github.com/freifunkMUC/meshviewer/blob/73eb86bc7e5ec6ca1917ea7cc2efda446d8c123a/.github/workflows/docker.yml
> 
>     * https://github.com/freifunkMUC/meshviewer/blob/73eb86bc7e5ec6ca1917ea7cc2efda446d8c123a/.github/workflows/build.yml
> 
>     * https://github.com/freifunkMUC/meshviewer/blob/7f168658a5eef54a2659406744a73fc688243837/docker-compose.yml

Sounds like a good idea. Should I do the upstream PR or will you do it?

**Comment by T0biii** [comment]
@GoliathLabs can you do the upstream pr pls :)

**Comment by GoliathLabs** [comment]
https://github.com/freifunk/meshviewer/pull/108

**Pull request merged**